pkg/storagesvc/client/storagesvc_test.go

```diff
@@ -57,10 +58,18 @@ func MakeTestFile(size int) *os.File {
 	return f
 }
 
+func checkArch() string {
+	if runtime.GOARCH == "arm64" {
+		return "RELEASE.2020-07-24T22-43-05Z-arm64"
```

Comment:
Can we use git commit / git commit + date to track? 
Something like [this](https://github.com/openebs/node-disk-manager/blob/master/build/build.sh#L20)?